listener/Dockerfile

```diff
@@ -140,6 +142,22 @@ ENV TCF_HOME=/project/avalon
 
 WORKDIR /project/avalon/listener
 
-COPY --from=build_image /usr/local/ /usr/local
 COPY --from=build_image /project/avalon/listener/listener_config.toml /project/avalon/listener/listener_config.toml
 COPY --from=build_image /project/avalon/config/tcs_config.toml /project/avalon/config/
+# Copy Python build artifacts
+COPY --from=build_image /project/avalon/common/python/dist/*.whl dist/
+COPY --from=build_image /project/avalon/common/crypto/dist/*.whl dist/
+COPY --from=build_image /project/avalon/sdk/dist/*.whl dist/
+COPY --from=build_image /project/avalon/listener/dist/*.whl dist/
+
+# Installing wheel file requires python3-pip package.
+# But python3-pip package will increase size of final docker image.
+# So remove python3-pip package and dependencies after installing wheel file.
+RUN apt-get update \
+ && apt-get install -y -q python3-pip \

```

Comment:
python3-pip package is already part of base_image. No need to install it again here.

Reply:
python json-rpc package may not be needed during build stage. This could be installed in final image instead of base image. so python-pip3 is not required to be installed in base image.

Reply:
Done